### PR TITLE
Fix for JRuby 9.2 or later

### PR DESCRIPTION
**Pull request by azumakuniyuki**
- It worked at JRuby 9.3
- #159 
- #267

### CODE REVIEW

File: .github/workflows/rake-test.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+jobs:
+  test-cruby:
+    name: rake test with CRuby ${{ matrix.cruby }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        cruby: ["2.4", "2.7", "3.0", "3.3"]
+    steps:
+      - uses: actions/checkout@v2
```
Comment (hiroyuki-sato) [inline .github/workflows/rake-test.yml]: minor: You will find the deprecated message. `checkout@v4` available. You just need to change v2 -> v4
https://github.com/actions/checkout

**Review by hiroyuki-sato — COMMENTED** [review]
@azumakuniyuki I left a minor comment. Please check when you get a chance.

File: .github/workflows/rake-test.yml (hunk 1 above)
Comment (azumakuniyuki) [reply .github/workflows/rake-test.yml]: @hiroyuki-sato Thanks for the review, I'll use v4 :-)

**Comment by azumakuniyuki** [comment]
CI works properly. #159 #267 #268 #269 #270

**Pull request merged**